packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -689,21 +689,31 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
      * @param validator Address of the validator.
      */
     function bondValidatorKro(address validator) external onlyValidatorManager {
-        Vault storage vault = _vaults[validator];
-        if (vault.asset.validatorKro - vault.asset.validatorKroBonded < BOND_AMOUNT)
-            revert InsufficientAsset();
+        Asset storage asset = _vaults[validator].asset;
+        if (asset.validatorKro - asset.validatorKroBonded < BOND_AMOUNT) revert InsufficientAsset();
```

Comment:
I think saving `asset.validatorKro - asset.validatorKroBonded` in memory variable like `remainingAssets` and using it at the bottom of this function like `emit ValidatorKroBonded(..., remainingAssets - BOND_AMOUNT)` can save some gas, wdyt?